source/agora/test/Base.d

```diff
@@ -1267,22 +1267,7 @@ public class TestNetworkManager : NetworkManager
     }
 
     ///
-    protected final override BlockExternalizedHandler getBlockExternalizedHandler
-        (Address address)
-    {
-        assert(0, "Not supported");
-    }
-
-    ///
-    protected final override PreImageReceivedHandler getPreimageReceivedHandler
-        (Address address)
-    {
-        assert(0, "Not supported");
-    }
-
-    ///
-    protected final override TransactionReceivedHandler getTransactionReceivedHandler
-        (Address address)
+    protected final override auto getHandler (HandleType)(Address address)
```

Comment:
The compiler should have screamed at you for using `override` here. Bad compiler.

Reply:
Reverted